src/sidebar/templates/login_control.html

```diff
@@ -20,7 +20,11 @@
   </a>
   <ul class="dropdown-menu pull-right" role="menu">
     <li class="dropdown-menu__row" ng-if="vm.auth.status === 'logged-in'">
-      <a href="{{vm.serviceUrl('user',{user: vm.auth.username})}}"
+      <span ng-if="vm.isThirdPartyUser()"
+         class="dropdown-menu__link dropdown-menu__link--disabled"
+         target="_blank">{{vm.auth.username}}</span>
```

Comment:
It does seem odd to have a menu item which looks like a link but doesn't go anywhere. We can review this collectively once everyone has had a chance to see it.